config_files/training/config_lorem_ipsum_span_masking_T5.yaml

```diff
@@ -0,0 +1,272 @@
+settings:  
+  experiment_id: ${modalities_env:experiment_id}
+  config_file_path: ${modalities_env:config_file_path}
+  referencing_keys:
+    sample_key: input_ids
+    target_key: target_ids
+  training:
+    training_log_interval_in_steps: 48
+    checkpointing_interval_in_steps: 48
+    evaluation_interval_in_steps: 48
+    global_num_seen_tokens: 0
+    do_apply_activation_checkpointing: false
+    gradient_acc_steps: 1
+    local_train_micro_batch_size: 2
+    sequence_length: 512
+  cuda_env:
+    local_rank: ${cuda_env:LOCAL_RANK}
+    global_rank: ${cuda_env:RANK}
+    world_size: ${cuda_env:WORLD_SIZE}
+  paths:
+    checkpointing_path: ./checkpoints
+
+tokenizer:
+  component_key: tokenizer
+  variant_key: pretrained_hf_tokenizer
+  config:
+    pretrained_model_name_or_path: google/long-t5-tglobal-base
+    padding: false
+    truncation: false
+
+collate_fn:  
+  component_key: collate_fn
+  variant_key: span_masking_collator_wrapper
+  config:
+    sample_key: ${settings.referencing_keys.sample_key}
+    target_key: ${settings.referencing_keys.target_key}
+    noise_density: 0.3
+    mean_noise_span_length: 3
+    tokenizer:
```

Comment:
need to check if we really need the tokenizer dependency

Reply:
Removed, now only pass vocab_size (int)